src/agent/src/rpc.rs

```diff
@@ -998,12 +999,24 @@ impl agent_ttrpc::AgentService for AgentService {
 
         info!(sl!(), "get_ip_tables: request received");
 
+        // the binary could exists in both /usr/sbin and /sbin
```

Comment:
`both /usr/sbin and /sbin` -> `either /usr/sbin or /sbin`?

Reply:
Fixed